spec/factories/vacancies.rb

```diff
@@ -79,7 +79,7 @@
     trait :expired do
       status { :published }
       sequence(:slug) { |n| "slug-#{n}" }
-      publish_on { Faker::Time.backward(14) }
+      publish_on { Faker::Time.between(Time.zone.today - 14.days, Time.zone.today - 7.days) }
       expires_on { Faker::Time.backward(7) }
```

Comment:
We should change this to 6 to ensure that the publication and expiry date don't overlap.

Reply:
@tahb I think this change should be applied to the expires_on, as now there is ac hance of the publication date being after the expiry date.